src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-additional-information/child-additional-information.js

```diff
@@ -0,0 +1,121 @@
+import { genericSchemas } from '../../../generic-schema';
+import { isChapterFieldRequired } from '../../../helpers';
+import { childInfo } from '../child-information/helpers';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    childrenToAdd: {
+      type: 'array',
+      minItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          doesChildLiveWithYou: {
+            type: 'string',
+            enum: ['Yes', 'No'],
+            default: 'Yes',
+          },
+          childAddressInfo: {
+            type: 'object',
+            properties: {
+              personChildLivesWith: {
+                type: 'object',
+                properties: {
+                  first: genericSchemas.genericTextInput,
+                  middle: genericSchemas.genericTextInput,
+                  last: genericSchemas.genericTextInput,
+                },
+              },
+              childAddress: {
+                type: 'object',
+                properties: {
+                  country: genericSchemas.genericTextInput,
+                  street: genericSchemas.genericTextInput,
+                  line2: genericSchemas.genericTextInput,
+                  line3: genericSchemas.genericTextInput,
+                  city: genericSchemas.genericTextInput,
+                  state: genericSchemas.genericTextInput,
+                  postal: genericSchemas.genericNumberAndDashInput,
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+};
+
+export const uiSchema = {
+  childrenToAdd: {
+    'ui:options': {
+      itemName: 'Child',
+      viewField: childInfo,
+    },
+    items: {
+      doesChildLiveWithYou: {
+        'ui:widget': 'radio',
+        'ui:title': 'Does this child live with you?',
+      },
+      childAddressInfo: {
+        'ui:options': {
+          expandUnder: 'doesChildLiveWithYou',
+          expandUnderCondition: 'No',
+          keepInPageOnReview: true,
+        },
+        personChildLivesWith: {
+          'ui:title': 'Person your child lives with',
+          first: {
+            'ui:title': 'First name',
+            'ui:required': formData =>
+              isChapterFieldRequired(formData, 'addChild'),
+          },
+          middle: {
+            'ui:title': 'Middle name',
+          },
+          last: {
+            'ui:title': 'Last name',
+            'ui:required': formData =>
+              isChapterFieldRequired(formData, 'addChild'),
+          },
+        },
+        childAddress: {
+          'ui:title': "Child's address",
+          country: {
+            'ui:title': 'Country',
```

Comment:
This is a super knit pick...but it's generally cleaner to have consistent quote conventions throughout a file when possible. Again..knit pick.

Reply:
I agree, I will fix this. Not sure how that ended up there.